### PR TITLE
UI Test Case Refactoring

### PR DESCRIPTION
**Pull request by tgqiang**
UI Test Cases previously have duplication of code for creating <code>test_app</code> instance. This has since been fixed for this branch.

### CODE REVIEW

File: test/test_app.py [hunk 1]
```diff
@@ -1,12 +1,266 @@
...
+    TABLE_HEADER_ACTIONS = '<th>Actions</th>'
+
+
+    def __init__(self):
+        self.middleware = None
+        self.test_app = None
+
+
+    def setUp(self):
+        # loads an 'app.py' fixture
```
Comment (nlzz22) [inline test/test_app.py]: documentation for methods should be using ''' instead of #

File: test/test_module_mounting_fixed.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                                        'bottom" title="Not Mounted"></span>'
+
+
+    def __init__(self):
+        self.middleware = None
+        self.test_app = None
+
+
+    def setUp(self):
+        # loads an 'app.py' fixture
```
Comment (nlzz22) [inline test/test_module_mounting_fixed.py]: documentation for method should use ''' instead of #

File: test/test_module_mounting_tentative.py [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+                                        'bottom" title="Not Mounted"></span>'
+
+
+    def __init__(self):
+        self.middleware = None
+        self.test_app = None
+
+
+    def setUp(self):
+        # loads an 'app.py' fixture
```
Comment (nlzz22) [inline test/test_module_mounting_tentative.py]: similar issue, use ''' instead of #

File: test/test_app.py [hunk 4]
```diff
@@ -50,15 +50,22 @@ class TestCode(object):
...
+        self.middleware = []
+        self.test_app = TestApp(APP.wsgifunc(*self.middleware))
+
+
```
Comment (a0129998) [inline test/test_app.py]: defined self.test_app during initialisation and setup to be reused.

**Comment by a0129998** [comment]
passed all 45 testcases in integration branch

**Pull request merged**